boxwise_flask/models/size.py

```diff
@@ -0,0 +1,47 @@
+from peewee import CharField, IntegerField
+
+from ..db import db
+
+max_int_size_for_my_sql_int = 2 ^ 32 - 1
```

Comment:
In Python, ^ is the operator for bitwise XOR. You probably want exponentation, that's the ** operator